packages/eslint-plugin/src/rules/atom-postifx-rule.ts

```diff
@@ -0,0 +1,97 @@
+import type { Rule } from 'eslint'
+import type { CallExpression, Identifier, Literal, Node } from 'estree'
+import {
+  extractAssignedVariable,
+  extractAssignedVariableName,
+  extractImportDeclaration,
+  isLiteral,
+  traverseBy,
+} from '../lib'
+import { isAtomCallExpression } from './atom-rule'
+
+type AtomCallExpression = CallExpression & {
+  callee: Identifier
+  arguments: [Literal] | [Literal, Literal]
+}
+
+export const atomPostfixRule: Rule.RuleModule = {
+  meta: {
+    type: 'suggestion',
+    docs: {
+      description: 'Check postfix for every atom',
+    },
+    fixable: 'code',
+    messages: {
+      incorrectVariableName: `atom "{{ atomName }}" should have postfix "{{ postfix }}"`,
```

Comment:
```suggestion
      incorrectVariableName: `atom "{{ atomName }}" should have postfix and be "{{ correctName }}"`,
```

Reply:
This message may be not really good if it will be
countAto -> countAtoAtom (but with postfix message it can be interpreted easier)